app/assets/javascripts/temperature.js

```diff
@@ -1,26 +1,25 @@
 $(function () {
   'use strict';
-
   window.Temperature = window.Temperature || {};
 
   Temperature.setData = function (widget, data) {
-    widget.find('.widget-hotness').removeClass('hotness0');
-    widget.find('.widget-hotness').removeClass('hotness1');
-    widget.find('.widget-hotness').removeClass('hotness2');
-    widget.find('.widget-hotness').removeClass('hotness3');
-    widget.find('.widget-hotness').removeClass('hotness4');
+    widget.find('.widget-temperature').removeClass('temperature0');
```

Comment:
`removeClass` can take many arguments, so this could be 

```js
widget.find('.widget-temperature').removeClass("temperature0", "temperature1", ...);
```